maps/src/MapBinner.cxx

```diff
@@ -18,7 +18,8 @@ class MapBinner : public G3Module {
 	MapBinner(std::string output_map_id, const G3SkyMap &stub_map,
 	    std::string pointing, std::string timestreams,
 	    std::string detector_weights, std::string bolo_properties_name,
-	    bool store_weight_map, boost::python::object map_per_scan);
+	    bool store_stokes_map, bool store_weight_map,
```

Comment:
I guess T is a stokes parameter, but it still seems a little odd as a name. Elsewhere we refer to the "map" and the "weight map". Can this be "store_map"?